cmd/loogtui/main.go

```diff
@@ -51,6 +51,15 @@ func init() {
 
 func main() {
 	flag.Parse()
+	if flagOutFile == "" {
+		file, err := os.CreateTemp("", "loog-output-*.loog")
```

Comment:
close file after reading and add defer to remove

Reply:
Added in 8ef054e6bf3d2ffb0f18f4102ae262bf4af76355

Reply:
defers are stacked in LIFO, meaning it would delete, then close the file